.setup/distro_setup/ubuntu/18.04/setup_distro.sh

```diff
@@ -82,11 +82,6 @@ apt-get install -qqy cmake
 # for Lichen (Plagiarism Detection)
 apt-get install -qqy python-clang-6.0
 
-# Install OpenJDK8 Non-Interactively
-echo "installing java8"
-apt-get install -qqy openjdk-8-jdk
-update-java-alternatives --set java-1.8.0-openjdk-amd64
```

Comment:
Let's leave Java for 16.04 and 18.04. 16.04 is deprecated and fully unsupported / unmaintained, and 18.04 is close enough to that status as well that better to just leave it as-is.